index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Shingirai Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <a href="/" class="header-logo">
+            <h3>
+                Welcome
+            </h3>
+        </a>
+        <nav>
+            <img src="./assets/Union.svg" alt="menu link">
+        </nav>
+    </header>
+    <section class="intro">
+        <h1>
+            Hey there. I’m Shingirai
+        </h1>
+        <h1>
+            <span>I’m a software developer</span>
+        </h1>
```

Comment:
- [ ] Kindly change this h1 to a p tag as this is part of the project requirements. It is better for your page to have just one h1 tag, this makes your website semantic and easily readable by browsers 👍🏿

Reply:
@prolajumokeoni 
**Thank you for pointing that out, I have changed to using one h1 tag using this method, not the p tag**
<h1>Hey there. I’m Shingirai<span><br>
         I’m a software developer</span></h1>
 **Because they are inline elements and they share the same attributes**

Reply:
Ok, you should use any header tag from h2 below, Good job 💪🏿